build.gradle

```diff
@@ -37,6 +37,13 @@ dependencies {
     implementation 'edu.wpi.first.thirdparty.frc2024.opencv:opencv-java:4.8.0-4'
 }
 
+// Setup Javadocs to link back to WPILib docs
+javadoc {
+    options {
+        links 'https://docs.oracle.com/en/java/javase/17/docs/api/', 'https://github.wpilib.org/allwpilib/docs/release/java/'
```

Comment:
Abstract out the version here and put it near where we set the compile version.

Reply:
At the top of this script or after the wpilib version change?

Reply:
https://github.com/wpilibsuite/vendor-template/blob/2025/build.gradle#L20

Here, so we know we need to change the docs version if we change the java target.